ipinfo/cmd_download.go

```diff
@@ -0,0 +1,149 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/ipinfo/cli/lib/complete"
+	"github.com/ipinfo/cli/lib/complete/predict"
+	"github.com/spf13/pflag"
+)
+
+const dbDownloadURL = "https://ipinfo.io/data/free/"
+
+var completionsDownload = &complete.Command{
+	Flags: map[string]complete.Predictor{
+		"-f":       predict.Nothing,
+		"--format": predict.Nothing,
+		"-t":       predict.Nothing,
+		"--token":  predict.Nothing,
+		"-h":       predict.Nothing,
+		"--help":   predict.Nothing,
+	},
+}
+
+func printHelpDownload() {
+	fmt.Printf(
+		`Usage: %s download [<database>] [<opts>]
+
+Description:
+    Download the free ipinfo databases.
+
+Examples:
+    # Download country database in csv format.
+    $ %[1]s download country -f csv
+
+Databases:
+    asn            free ipinfo asn database.
+    country        free ipinfo country database.
+    country-asn    free ipinfo country-asn database.
+
+Options:
+  General:
+    --token <tok>, -t <tok>
+      use <tok> as API token.
+    --help, -h
+      show help.
+
+Outputs:
+    --format , -f <mmdb | json | csv>
+    output format of the database file.
+      mmdb (default.)  downloads the mmdb format database.
+      json             downloads the json format database.
+      csv              downloads the csv  format database.
```

Comment:
Options are already presented now in `<>` and explanations are redundant. Just need to identify the default in the same way we do elsewhere.

```suggestion
      output format of the database file.
      default: mmdb.
```

Reply:
Also note the spacing issue.